guides/common/modules/proc_disabling-dhcp-for-integration.adoc

```diff
@@ -0,0 +1,42 @@
+[id="disabling-dhcp-for-integration"]
+= Disabling DHCP for integration
+
+If you want to manually manage a DHCP service, you must prevent {Project} from maintaining this service on the operating system and disable orchestration to avoid errors.
+
+[NOTE]
+====
+Disabling DHCP in {Project} does not remove the related backend service on the operating system.
+====
+
+
+.Prerequisites
+
+* You are logged in to the {ProjectWebUI}.
+
+
+.Procedure
+
+. Navigate to *Infrastructure* > *Subnets*.
+
+. For each subnet that is associated with the DHCP {SmartProxy}:
+
+.. Select the subnet.
+
+.. On the *{SmartProxies}* tab, clear the *DHCP {SmartProxy}* field.
+
+.. Click *Submit*.
+
+. On {ProjectServer} and {SmartProxyServer}, enter:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+# {foreman-installer} --foreman-proxy-dhcp false
+----
+
+
+[NOTE]
+====
+{Project} does not perform orchestration when a {SmartProxy} is not set for a given subnet.
+When you disable {SmartProxy} associations, orchestration commands for existing hosts can fail if the expected records and configuration files are not present.
+====
```

Comment:
Thinking of the possible DITA migration: I think free hanging paragraphs or notes are not supported, are they? Can you find a way to integrate this into the intro, for example?

Reply:
Applies elsewhere too.

Reply:
The admonition should be connected to the last step. This is fixed now. Same in the other procedure.